example/go.sum

```diff
@@ -1,14 +1,12 @@
-github.com/99designs/gqlgen v0.17.20 h1:O7WzccIhKB1dm+7g6dhQcULINftfiLSBg2l/mwbpJMw=
```

Comment:
[dependency] The following dependencies have been updated: github.com/99designs/gqlgen from v0.17.20 to v0.17.31, github.com/agnivade/levenshtein from v1.1.1 to v1.0.1, github.com/davecgh/go-spew from v1.1.0 to v1.1.1, github.com/go-logr/logr from v1.2.3 to v1.2.4, github.com/go-logr/stdr from v1.2.1 to v1.2.2, github.com/google/go-cmp from v0.5.5 to v0.5.9, github.com/gorilla/websocket from v1.4.2 to v1.5.0, github.com/hashicorp/golang-lru from v0.5.4 to v2.0.1, github.com/kevinmbeaulieu/eq-go from v1.0.0 to v1.0.1.